setup.sh

```diff
@@ -147,6 +150,12 @@ if ! ${PYTHON} -m pip -V; then
     ${PYTHON} -m ensurepip ${INSTALLFLAGS} --upgrade
 fi
 
+if [ "$1" == "virtualenv" ]
```

Comment:
I'm pretty sure that your `venv` install mechanism does not ACTUALLY use virtualenv.  `${INSTALLFLAGS}` uses the same hard-coded path as GDB uses.